packages/components/src/MultiSelect/MultiSelect.spec.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from "react"
-import { render, waitFor } from "@testing-library/react"
+import { render, waitFor, screen } from "@testing-library/react"
```

Comment:
Using screen here since this is the decided way going forward but will hold off on refactoring the others to use this to a later time